tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/StorageAnimalList.java

```diff
@@ -0,0 +1,66 @@
+package com.sci.bv.tema14Ianuarie;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class StorageAnimalList {
+
+    ArrayList<AnimalProduct> list = new ArrayList<AnimalProduct>();
+
+    public StorageAnimalList() {
+    }
+
+    /**
+     * This method adds animal products in stock in the ArrayList "AnimalProduct",
+     * together with it's characteristics.
+     *
+     * @return a new product in stock
+     */
+    public ArrayList<AnimalProduct> addAnimalProduct() {
```

Comment:
maybe "addAnimalProductFromConsole()" otherwise I would expect to receive the AnimalProduct as a parameter.
Why are you returning the entire list of animal products here? Can't someone query the Storage later on with a getAnimalProducts method?